server/app/models/UserModel.ts

```diff
@@ -24,7 +24,6 @@ export const UserSchema: Schema = new Schema({
   password: {
     type: Schema.Types.String,
     required: true,
-    validate: [validator.isStrongPassword, 'Password must contain a capital, symbol and a number.'],
```

Comment:
is this one brokey or we prefer to validate in the service.

Reply:
nah i was just being stupid, the password is hashed before the model validation :facepalm: So i need to validate before hashing in the service layer